hw/data/common_project_cfg.hjson

```diff
@@ -8,8 +8,10 @@
   results_server:   reports.opentitan.org
 
   // Default directory structure for the output
-  scratch_base_path:  "{scratch_root}/{branch}"
-  scratch_path:       "{scratch_base_path}/{dut}-{flow}-{tool}"
+  scratch_base_path:   "{scratch_root}/{branch}"
+  scratch_path:        "{scratch_base_path}/{dut}-{flow}-{tool}"
+  report_summary_path: "{scratch_root}/{branch}/report/summary"
+  report_path:         "{scratch_root}/{branch}/report/{dut}-{flow}-{tool}"
```

Comment:
Hmm, now that I think about it more, it seems like we could simply reuse the `scratch_path` to find the HTML report as well. Thoughts?

Reply:
true, then we can avoid adding another variable. I just add it here cause then I can use the `.mk` file to create each folder for reports (I mean the folder: `{scratch_root}/{branch}/report/{dut}-{flow}-{tool}`)
I do not have any strong preference, if using `scratch_path`, we can avoid this variable, and use a for loop to create each folder in `dvsim` :)

Reply:
Yes, that's right. Use a fixed directory called `report` under `scratch_path` to put the generated HTML and all other report files.